src/config.sh

```diff
@@ -244,5 +312,6 @@ function config_help()
   printf '%s\n' 'kw config <config.option value>:' \
     '  config - Show config values' \
     '  config (-g | --global) <config.option value> - Change global config' \
-    '  config (-l | --local) <config.option value> - Change local config'
+    '  config (-l | --local) <config.option value> - Change local config' \
+    '  config (-s | --show) [config]... - Show all or specific current configurations'
```

Comment:
You also need to update the man documentation and maybe add this to some of the tutorials.